tsl/test/sql/dist_hypertable.sql.in

```diff
@@ -1562,3 +1562,47 @@ SET timescaledb.enable_per_data_node_queries = true;
 \set ON_ERROR_STOP 0
 SELECT tableoid::regclass, * FROM dist_syscol;
 \set ON_ERROR_STOP 1
+
+-- Test SkipScan
+-- Add an index on the value column
```

Comment:
can you add a test with hashagg disabled on the datanode, wanna see a plan with unique on datanode

Reply:
@svenklemm did you see the tsl/test/expected/dist_hypertable-13.out as an example? Almost all datanode `SELECT DISTINCT` queries have a `Unique` on top. Example:

```
->  Custom Scan (DataNodeScan) on public.hyper hyper_3     
                           Output: hyper_3."time", hyper_3.device               
                           Data node: db_dist_hypertable_3                      
                           Chunks: _dist_hyper_27_82_chunk, _dist_hyper_27_85_chunk
                           Remote SQL: SELECT DISTINCT ON (device) "time", device FROM public.hyper WHERE _timescaledb_internal.chunks_in(public.hyper.*, ARRAY[31, 32]) ORDER BY device ASC NULLS LAST
                           Remote EXPLAIN:                                      
                             Unique                                             
                               Output: _dist_hyper_27_82_chunk."time", _dist_hyper_27_82_chunk.device
                               ->  Merge Append                                 
                                     Sort Key: _dist_hyper_27_82_chunk.device   
                                     ->  Custom Scan (SkipScan) on _timescaledb_internal._dist_hyper_27_82_chunk
                                           Output: _dist_hyper_27_82_chunk."time", _dist_hyper_27_82_chunk.device
```

Reply:
oh sorry i meant access node

Reply:
Ok, added test to check use of `Unique` on access node